libs/features/mas/.hlxignore

```diff
@@ -0,0 +1,7 @@
+.*
```

Comment:
what about /libs/features/mas/**/* in root hlxignore?

Reply:
Added `*` to `.hlxognore` in root folder of `features/mas` as requested.

Reply:
addressed